packages/docs/src/Header.tsx

```diff
@@ -26,6 +34,14 @@ const HeaderContainer = styled.div`
   align-items: center;
 `
 
+const ThemeWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  ${Typography} {
+    margin-right: ${spacing.medium};
+  }
```

Comment:
Or grid + gap, or flex gap is almost usable as well.

Reply:
`gap`? You learn something new every day. Fixed.